pkg/controllers/klusterlet/manifests/klusterlet/management/klusterlet-agent-deployment.yaml

```diff
@@ -26,10 +26,15 @@ spec:
           - "--cluster-name={{ .ClusterName }}"
```

Comment:
Do we start this deployment?

Reply:
the klusterlet controller will apply this to the management cluster to deploy the agent with hosted mode